src/CMakeLists.txt

```diff
@@ -1658,6 +1658,16 @@ SET(VISIT_COMMON_INCLUDES
     ${ZLIB_INCLUDE_DIR}
     )
 
+if(WIN32)
```

Comment:
Install logic for third party libraries should be handled in their Find module. For VisIt's find modules, install is generally handled via the call to SET_UP_THIRD_PARTY.

Reply:
I'll have to review what this was thought to be necessary.  Perhaps nothing along these lines is needed and I can then remove this.  Will try after all issues resolved.